code-samples/oauth/simple-oauth-webserver/index.html

```diff
@@ -20,18 +20,17 @@
 			return randStr;
 		}
 
-		window.onload = () => {
-			const fragment = new URLSearchParams(window.location.hash.slice(1));
-
-			if (fragment.has("access_token")) {
-				const urlState = fragment.get("state");
+		window.onload = function () {
```

Comment:
```suggestion
		window.onload = () => {
```
why was this changed to `function`?